package.json

```diff
@@ -158,5 +158,8 @@
     "lint-staged": {
         "*.{js,css,md,less,json}": "prettier --write",
         "*.{css,less}": "stylelint --config .stylelintrc"
+    },
+    "overrides": {
+        "clean-css": "^5.3.3"
```

Comment:
@ArtBlue this was the update that allowed me to use container queries

Reply:
Thanks, @mikehobi !!